src/core.js

```diff
@@ -100,6 +100,9 @@ config = {
 	// by default, scroll to top of the page when suite is done
 	scrolltop: true,
 
+	// By default, only expand the first failed test as opposed to all failed tests
+	collapse: true,
```

Comment:
Making this configurable seems sane, though based on the comment its unclear what `collapse: false` would do - collapse nothing? Collapse all tests?